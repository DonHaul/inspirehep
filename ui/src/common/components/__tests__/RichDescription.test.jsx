import { render } from '@testing-library/react';

import RichDescription from '../RichDescription';

describe('RichDescription', () => {
  it('renders with description', () => {
    const { asFragment } = render(
      <RichDescription>description</RichDescription>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
