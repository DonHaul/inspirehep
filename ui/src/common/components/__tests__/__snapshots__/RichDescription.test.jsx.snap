// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RichDescription renders with description 1`] = `
<DocumentFragment>
  <div>
    description
  </div>
</DocumentFragment>
`;
